data/fixtures/scopes/textual/boundedParagraph5.scope

```diff
@@ -0,0 +1,132 @@
+{aaa
+
+bbb
+
+ccc}
+---
+
+[#1 Content] =
+[#1 Domain] = 0:0-0:4
+  >----<
+0| {aaa
+
+[#1 Removal] = 0:0-2:0
+  >----
+0| {aaa
+1|
+2| bbb
+   <
+
+[#1 Trailing delimiter: Content] = 1:0-1:0
+  ><
+1|
+[#1 Trailing delimiter: Removal] = 1:0-2:0
+  >
+1|
+2| bbb
+   <
+
+[#1 Insertion delimiter] = "\n\n"
+
+
+[#2 Content] =
+[#2 Domain] = 0:1-0:4
+   >---<
+0| {aaa
+
+[#2 Removal] = 0:1-1:0
+   >---
+0| {aaa
+1|
+   <
```

Comment:
Arguably this should extend to the beginning of `bbb`, but I can't imagine this edge case is too common